### DISCUSSION TITLE
glide-data-grid || Header Cell Styling

### DISCUSSION THREAD

**Discussion by NJ007-njp** [glideapps/glide-data-grid#273] [Ideas]
Does Glide-data-table allow to style header cell, such as padding, white-space-break, apart from applying theming?

I have requirement such that, in header date need to be shown in following format :

wed
04/
20

where as in cell it is displayed as wed 04/ 20

**Comment by jassmith** [comment]
This is possible by overriding the rendering of the cell header. It's definitely not as easy as doing it with css but it can be done.

**Reply by NJ007-njp** [reply]
drawHeader?: (args: {
    ctx: CanvasRenderingContext2D;
    column: GridColumn;
    theme: Theme;
    rect: Rectangle;
    hoverAmount: number;
    isSelected: boolean;
    isHovered: boolean;
    hasSelectedCell: boolean;
    spriteManager: SpriteManager;
    menuBounds: Rectangle;
}) => boolean;


which property need to be modified, in order to apply break on white space?

**Reply by jassmith** [reply]
This is canvas, one does not simply get whitespace breaks for free. You have to take basically make a copy of the current header draw code but issue a different drawText command for each line of text. I will write you a simple example sometime this week.
